README.md

```diff
@@ -79,6 +79,12 @@ Deploying Chatwoot to Heroku is a breeze. It's as simple as clicking this button
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/chatwoot/chatwoot/tree/master)
 
+###### Post Deployment
```

Comment:
Hey @SmartManoj, can you please move this section to the following file?

https://github.com/chatwoot/chatwoot/blob/develop/docs/deployment/production/heroku.md

and please download the image and create a folder `images` inside `docs/deployment/production/` and use that reference inside.

Instead of this section, can you please add a link to the detailed guide?